Android version update & misc fixes; credit to @janisozaur for initial PR (#356)
microsoft/vcpkg-docs: vcpkg/users/platforms/android.md

---

> title: Android support in vcpkg

Reviewer: ```suggestion
title: Android support
```

   Or:

   ```bash

>    export ANDROID_NDK_HOME=/home/your-account/Android/android-ndk-r26d

Reviewer: Can we find a way to describe this without naming the NDK version?

If you prefer using [the latest version](https://developer.android.com/studio/projects/install-ndk#default-ndk-per-agp), check the [BuildSystemMaintainers.md of the NDK document](https://android.googlesource.com/platform/ndk/+/master/docs/BuildSystemMaintainers.md#sysroot) and then put appropriate path for your system.

> For example, macOS users will use the path like this example:

Reviewer: ```suggestion
For example, MacOS users will use the path like this example:
```
Still a Proper Noun, iiuc

[!INCLUDE [experimental](../../../includes/experimental.md)]

> vcpkg can export Android archives ([AAR files](https://developer.android.com/studio/projects/android-library)). Once an archive is created, it can imported in Android Studio as a native dependent.  The archive is automatically consumed using [Android Studio's prefab tool](https://github.com/google/prefab).

Reviewer 2: ```suggestion
vcpkg can export Android archives ([AAR files](https://developer.android.com/studio/projects/android-library)). Once an archive is created, it can be imported in Android Studio as a native dependent.  The archive is automatically consumed using [Android Studio's prefab tool](https://github.com/google/prefab).
```

### Example exporting \[jsoncpp]

> First, "vcpkg install" the 4 Android architectures (it is mandatory to export all 4 of them):

Reviewer 2: ```suggestion
First, run "vcpkg install" for all supported Android architectures (it is mandatory to export all of them):
```
We probably want to get rid of the specific number in case it changes.
